x-pack/test/defend_workflows_cypress/agent.ts

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import os from 'node:os';
+import execa from 'execa';
+import { find } from 'lodash';
+import { ToolingLog } from '@kbn/tooling-log';
+import axios, { AxiosRequestConfig } from 'axios';
+import { ChildProcess } from 'child_process';
+// import { getLatestVersion } from './artifact_manager';
+import { Manager } from './resource_manager';
+
+export interface AgentManagerParams {
+  user: string;
+  password: string;
+  kibanaUrl: string;
+  esHost: string;
+  esPort: string;
+}
+
+export class AgentManager extends Manager {
+  private params: AgentManagerParams;
+  private log: ToolingLog;
+  private agentProcess?: ChildProcess;
+  private requestOptions: AxiosRequestConfig;
+  private vmName: string;
+  private agentPolicyId?: string;
+  constructor(params: AgentManagerParams, log: ToolingLog, requestOptions: AxiosRequestConfig) {
+    super();
+    this.log = log;
+    this.params = params;
+    this.requestOptions = requestOptions;
+    this.agentPolicyId = undefined;
+    this.vmName = Math.random().toString(36).slice(2);
+  }
+
+  public async setup() {
+    this.log.info('Running agent preconfig');
+
+    const {
+      data: { items: packages },
+    } = await axios.post(
+      `${this.params.kibanaUrl}/api/fleet/epm/packages/_bulk`,
```

Comment:
can we use Fleet services to get the paths here and in other areas of this PR that reference these APIs?  They make available a set of services that you can use here:

https://github.com/elastic/kibana/blob/f71af91b9c6f821baf1123c2d235010d2d140686/x-pack/plugins/fleet/common/services/routes.ts#L66-L68